release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 8. | **Feature** | Support for Managed KeyVaultReference | **Brief Description**: Service Fabric's auto-rotation story for application secrets is here: Managed KeyVaultReferences. Leverage Managed Identities for SF Apps, and KeyVaultReferences to get automatic application parameter upgrades as your secrets rotate in the vault.<br>**Documentation Reference**: Read more about it at - <ul><li>[Use KeyVaultReferences in your application](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-keyvault-references#use-keyvaultreferences-in-your-application), and </li><li>[Use Managed KeyVaultReferences in your application](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-keyvault-references#use-managed-keyvaultreferences-in-your-application)</li></ul> |
+| 9. | **Feature** | Expose Container ID for currently deployed code packages | **Brief Description**: For processes, Process ID is available from CodePackageEntryPoint structure in both Service Fabric Explorer (SFX) and through Get-ServiceFabricDeployedCodePackage in Powershell. However, for containers, the Container ID is not available via the same mechanism. We can still get this information but it requires multiple docker/containerD API calls to figure out which service fabric service/code package a particular container 'belongs' to. Service Fabric now exposes Container ID in the same fashion that Process ID is exposed.<br>**Solution**: Container ID is added to the CodePakageEntryPoint structure, exposing it in both Service Fabric Explorer (SFX) and through Get-ServiceFabricDeployedCodePackage in Powershell.<br>**Workaround**: Multiple queries to docker/containerD API to get the container ID and decide which is it's corresonding code package. |
```

Comment:
However, for containers, the Container ID is not available via the same mechanism. We can still get this information but it requires multiple docker/containerD API calls to figure out which service fabric service/code package a particular container 'belongs' to. Service Fabric now exposes Container ID in the same fashion that Process ID is exposed.

-> However, for containers the Container ID was not previously available. Service Fabric will now expose Container ID in the same fashion that Process ID is exposed.